src/lib/Room/RoomFooter/RoomFooter.vue

```diff
@@ -58,9 +58,12 @@
       class="vac-box-footer"
       :class="{ 'vac-box-footer-border': !files.length }"
     >
-      <div v-if="!isRecording && !textareaHighlight" class="vac-icon-textarea-left">
+    <template v-if="showAttachmentLoader">
+      <loader v-if="showAttachmentLoader" show="true" type="call-link-loader" flex="true" />
+    </template>
+      <div v-if="!isRecording && !textareaHighlight && !showAttachmentLoader" class="vac-icon-textarea-left">
```

Comment:
Como o **loader** e o "**mais de anexar**" nunca vão parecer na tela ao mesmo tempo, é sempre um **ou** o outro, aqui tu pode usar um `if` ... `else-if` e remover os `showAttachmentLoader` que estão depois do 1º `if`. Creio que ficaria assim:

```suggestion
    <template v-if="showAttachmentLoader">
      <loader show="true" type="call-link-loader" flex="true" />
    </template>
      <div v-else-if="!isRecording && !textareaHighlight" class="vac-icon-textarea-left">
```

Reply:
De fato.